### ISSUE TITLE
Data fliter and sorting 

### ISSUE THREAD

**Issue by chrislindor** [patrickkunka/mixitup#275]
Hey guys im having an issue with filtering the content with a specific sort oder with the same button. ive looked through closed issues and came across this one https://github.com/patrickkunka/mixitup/issues/129 by the solution leads to a dead link.

**Comment by patrickkunka** [comment]
Hi Chris,

Could you a post a reduced JSFiddle demonstrating the issue you're having?

**Comment by chrislindor** [comment]
[https://jsfiddle.net/h1c342at/](https://jsfiddle.net/h1c342at/) So i have a bunch of users and want to that belong to different groups. When i click on the filter i want it to also change the sort order of that specific group . But making a button activate a filter and sort order isnt working.

**Comment by patrickkunka** [comment]
Hi Chris,

I've got your fiddle working here:

https://jsfiddle.net/h1c342at/2/

Let me know if that's your intended behavior. You didn't seem to have any sort of constant target selector in your markup so I just added the standard classname 'mix' to both elements.

Also you were referencing MixItUp 2 (this may be because you couldn't find a CDN for v3), so I've updated that to MixItUp 3 as multimix buttons are a new feature in v3. I also added a container element on which to instantiate the mixer.
